Healthy/Classes/Modules/Login/LoginViewModel.swift

```diff
@@ -35,17 +35,16 @@ extension LoginViewModel: LoginViewModelOutput {
     var isLoadingIndicatorPublisher: AnyPublisher<Bool, Never> {
         $isLoadingState.eraseToAnyPublisher()
     }
-    
+
     var isShowErrorMessagePublisher: AnyPublisher<String, Never> {
```

Comment:
```suggestion
    var errorMessagePublisher: AnyPublisher<String, Never> {
```

is does reflect a boolean not a noun. If this is out of the scope, please add todo.